client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -289,24 +284,24 @@
                           @updateKind="handleFieldKindUpdate(index)"
                         />
                       </draggable>
+                    </b-table-simple>
+                  </c-form-table-wrapper>
+                </b-row>
 
-                      <tr>
-                        <td colspan="1" />
-                        <td colspan="7">
-                          <b-button
-                            data-test-id="button-field-add"
-                            class="mb-5"
-                            variant="primary"
-                            @click="handleNewField"
-                          >
-                            + {{ $t('edit.newField') }}
-                          </b-button>
-                        </td>
-                      </tr>
+                <b-row
+                  no-gutters
+                  class="mt-3"
+                >
+                  <b-table-simple
+                    borderless
```

Comment:
I think we should have columns like the above table and it should also be wrapped inside the wrapper